terminal/command.go

```diff
@@ -724,7 +726,11 @@ func printcontext(state *api.DebuggerState) error {
 	}
 	if len(state.CurrentThread.File) == 0 {
 		fmt.Printf("Stopped at: 0x%x\n", state.CurrentThread.PC)
-		fmt.Printf("\033[34m=>\033[0m    no source available\n")
+		if dumbTerminal {
```

Comment:
I have an idea for a refactor here.

Let's change all `fmt.Printf` calls to `term.Println(fmt.Sprintf("..."), true)` where the signature is `Term.Println(format string, currentLine bool)` and let's store the state of whether this is a dumb terminal or not in `Term`. Then, this function will turn into `printcontext(term, state)`. We have to do some work to pass the terminal into commands in `terminal/commands.go`, but I think overall it will be cleaner. Possibly instead of expanding the signature for commands in terminal/commands.go we can create a new type which will be the only argument to those functions, and that way it can be expanded in the future if need be without changing a lot of code.

The argument `currentLine` above will determine whether the arrow is printed. Also, instead of continuing to hard code the escape codes it would be best to at the very least store in a variable, or pluck them from https://github.com/golang/crypto/blob/master/ssh/terminal/terminal.go#L24 by just importing that terminal package and using the pre-defined escape codes there.

Thoughts?